decidim-generators/lib/decidim/generators/app_templates/initializer.rb

```diff
@@ -489,6 +489,12 @@
   end
 end
 
+if Decidim.module_installed? :verifications
+  Decidim::Verifications.configure do |config|
+    config.document_types = Rails.application.secrets.dig(:verifications, :document_types).presence || %w(identification_number passport)
```

Comment:
I had to change this to

```suggestion
    config.document_types = Rails.application.secrets.dig(:decidim, :verifications, :document_types).presence || %w(identification_number passport)
```

Reply:
So that it pulling from ```default; decidim```?

Reply:
@andreslucena I have reverted this change in commit: https://github.com/decidim/decidim/pull/12306/commits/2804bc1f19eec3f082570cabf1600b28dbad148c as it was a cause of CLI errors in the Verifications module. I also noticed the other modules in initalizers do not follow the same pattern. 

Perhaps you can try replicate this change locally to see if there are any errors?

Reply:
@greenwoodt if that's the case, then I think you need to check out the failing specs to actually solve them. As far as I could see, without the `decidim` symbol it doesn't work, as the key actually is "decidim.verifications.document_types", so doing "verifications.document_types" will never work and will not leave implementers to define their own document types:  

```console
$ bin/rails runner "puts Rails.application.secrets.dig(:verifications, :document_types).presence"
Running via Spring preloader in process 1545458

$ bin/rails runner "puts Rails.application.secrets.dig(:decidim, :verifications, :document_types).presence"
Running via Spring preloader in process 1545751
dni
nie
passport
```

Reply:
Right so why doesn't **Elections** follow the same pattern? 

```if Decidim.module_installed? :elections
  Decidim::Elections.configure do |config|
    config.setup_minimum_hours_before_start = Rails.application.secrets.dig(:elections, :setup_minimum_hours_before_start).presence || 1
    config.start_vote_maximum_hours_before_start = Rails.application.secrets.dig(:elections, :start_vote_maximum_hours_before_start).presence || 6
    config.voter_token_expiration_minutes = Rails.application.secrets.dig(:elections, :voter_token_expiration_minutes).presence || 120
    config.document_types = Rails.application.secrets.dig(:elections, :document_types).presence || %w(identification_number passport)
  end
```

Reply:
Pipline is green. Thanks!